domains/transfer/Transfer.API/API/UnitOfWork/IUnitOfWork.cs

```diff
@@ -0,0 +1,57 @@
+using System.Threading.Tasks;
+using API.Transfer.Api.Repository;
+using DataContext;
+using EnergyOrigin.ActivityLog.API;
+
+namespace API.UnitOfWork;
+
+public interface IUnitOfWork
```

Comment:
Shouldnt there be a rollback if any of these goes wrong? 
Have you read? https://learn.microsoft.com/en-us/archive/msdn-magazine/2009/june/the-unit-of-work-pattern-and-persistence-ignorance

Why is this implementation of unitofwork necessary compared to plain old DI? :)

Reply:
Because here we can do cross repo transactions. We can't with DI

Reply:
I havn't implemented rollback. THought that could be done when needed but can do it upfront if you want?